src/screens/MainScreen/index.js

```diff
@@ -28,11 +32,34 @@ const MainScreen = () => {
             provider={PROVIDER_GOOGLE}
             style={styles.map}
             region={{
-              latitude: 37.78825,
-              longitude: -122.4324,
+              latitude: userHasLocation ? latitude : 37.78825,
+              longitude: userHasLocation ? longitude : -122.4324,
```

Comment:
Maybe you can move these magic numbers to a constant to have a little bit more of semantic, like:
```js
const DEFAULT_REGION_COORDS = {
  latitude: 37.78825,
  longitude: -122.4324
};
```